python/cog/server/http.py

```diff
@@ -66,20 +67,8 @@ class Health(Enum):
     DEFUNCT = auto()
 
 
-class MyState:
-    health: Health
-    setup_result: Optional[SetupResult]
```

Comment:
Is this not still needed?

Reply:
`FastAPI` instances have a `state: State` property. I haven't found a way to make that typed, so it's instead an opaque object with getters and setters. To the point of the comment, the way we had it before broke type invariance and shouldn't have worked (by some definition of should).

I'm relieved that your refactoring of runner / worker reduced how much state we were tracking. For just a pair of properties, it's not too bad.

Reply:
My editor typechecker was understanding these types fine, though? Are gaining something by removing it other than a bit of neatness?